Controls/BootstrapDatepicker/src/DotVVM.Contrib/DotvvmConfigurationExtensions.cs

```diff
@@ -0,0 +1,70 @@
+﻿using DotVVM.Framework.Configuration;
+using DotVVM.Framework.ResourceManagement;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace DotVVM.Contrib
+{
+    public static class DotvvmConfigurationExtensions
```

Comment:
Because we're using same namespace as other dotvvm contrib controls (`DotVVM.Contrib`), this method should be named differently in order to avoid ambiguities.